### PR TITLE
[ML] Add documentation for post calendar events API

### PR DESCRIPTION
**Pull request by valeriy42**
This PR updates the documentation for the extension of the POST calendar events API implemented in #112837.

### CODE REVIEW

File: docs/reference/ml/anomaly-detection/apis/post-calendar-event.asciidoc [hunk 1]
```diff
@@ -54,6 +54,22 @@ milliseconds since the epoch or ISO 8601 format.
 `start_time`:::
 (Required, date) The timestamp for the beginning of the scheduled event in
 milliseconds since the epoch or ISO 8601 format.
+
+`skip_results`:::
+(Optional, Boolean) If `true`, the results durint the scheduled event are not created.
```
Comment (davidkyle) [inline docs/reference/ml/anomaly-detection/apis/post-calendar-event.asciidoc:59]:
```suggestion
(Optional, Boolean) If `true`, the results during the scheduled event are not created.
```

File: docs/reference/ml/anomaly-detection/apis/post-calendar-event.asciidoc [hunk 2]
```diff
@@ -54,6 +54,22 @@ milliseconds since the epoch or ISO 8601 format.
...
+(Optional, Boolean) If `true`, the model is not updated during the scheduled event.
+The default value is `true`.
+
+`force_time_shift`:::
+(Optional, integer) Allows you to shift the time within the anomaly detector
+by a specified number of seconds in a specified direction. This is useful to quickly
+adjust to known Daylight Saving Time (DST) events. For example, to account for a
+one-hour backward time shift during the fall DST event, use a value of `-3600`.
+The default value is `0`. The time is shifted once at the beginning of the scheduled
+event.
```
Comment (davidkyle) [inline docs/reference/ml/anomaly-detection/apis/post-calendar-event.asciidoc:72]: Make the fact that shift is in seconds explicit. 

```suggestion
event. The shift is measured in seconds.
```

File: docs/reference/ml/anomaly-detection/apis/post-calendar-event.asciidoc [hunk 3]
```diff
@@ -54,6 +54,22 @@ milliseconds since the epoch or ISO 8601 format.
...
+`skip_model_update`:::
+(Optional, Boolean) If `true`, the model is not updated during the scheduled event.
+The default value is `true`.
+
+`force_time_shift`:::
+(Optional, integer) Allows you to shift the time within the anomaly detector
+by a specified number of seconds in a specified direction. This is useful to quickly
+adjust to known Daylight Saving Time (DST) events. For example, to account for a
+one-hour backward time shift during the fall DST event, use a value of `-3600`.
+The default value is `0`. The time is shifted once at the beginning of the scheduled
```
[the remark this answers is not in the document]
Comment (valeriy42) [reply docs/reference/ml/anomaly-detection/apis/post-calendar-event.asciidoc]: Good catch Dan! I changed the sentence in [cb3a08e](https://github.com/elastic/elasticsearch/pull/113188/commits/cb3a08ecaaa02a6a9f91ed6a6d43752180b0c6b9).

**Review by szabosteve — APPROVED** [review]
Thank you for writing the docs! LGTM!

**Pull request merged**